lib/show_n_tell/github/authenticator.ex

```diff
@@ -0,0 +1,73 @@
+defmodule ShowNTell.Github.Authenticator do
+  alias ShowNTell.Repo
+  alias ShowNTell.User
+  alias ShowNTell.Rest.GithubApi
+  
+  # Authenticates a user and returns the user record.
+  def process_token({:ok, %{"access_token" => access_token}}) do
+    client = GithubApi.client(access_token)
+  
+    client
+    |> GithubApi.user_orgs
+    |> check_github_org
```

Comment:
Elixir style guide recommends using parentheses for one-arity functions https://github.com/christopheradams/elixir_style_guide#parentheses-pipe-operator